iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/AbstractConnectableXml.java

```diff
@@ -216,7 +216,12 @@ public static void writeCurrentLimits(Integer index, CurrentLimits limits, XMLSt
 
     public static void writeCurrentLimits(Integer index, CurrentLimits limits, XMLStreamWriter writer, IidmXmlVersion version,
                                           ExportOptions exportOptions) throws XMLStreamException {
-        writeCurrentLimits(index, limits, writer, version.getNamespaceURI(), version, exportOptions);
+        writeCurrentLimits(index, limits, writer, version, true, exportOptions);
+    }
+
+    public static void writeCurrentLimits(Integer index, CurrentLimits limits, XMLStreamWriter writer, IidmXmlVersion version,
+                                          boolean valid, ExportOptions exportOptions) throws XMLStreamException {
+        writeCurrentLimits(index, limits, writer, version.getNamespaceURI(valid), version, exportOptions);
```

Comment:
It is necessary to pass the "valid" flag in order to obtain the correct namespace. If it is not passed, and an attempt is made to export a network with SCADA level in ValidationLevel, it ends with an exception.

Reply:
Thank you for yor feedback! I think the methods used in the export is using the valid boolean while writing current limits, the only methods not using it are used elsewhere in always valid networks. I will check however!

Reply:
Fixed by d5fbedb849f2598af1d43e68df06515b50a76c06